src/functionalTest/java/uk/gov/hmcts/reform/sendletter/FunctionalTestSuite.java

```diff
@@ -172,7 +170,7 @@ protected String getFileNamePattern(String letterId) {
 
         return String.format(
             format,
-            Pattern.quote(SMOKE_TEST_LETTER_TYPE),
+            Pattern.quote("smoketest"),
```

Comment:
did the test pass before???

Reply:
I had tested it by posting the functional tests to sandbox but the service was updated after that.This is first time we tested on prod pipeline.If the file name generation logic was unchanged then we it would pass :)